### DISCUSSION TITLE
I can't make a new Repository

### DISCUSSION THREAD

**Discussion by SkyKingPX** [community/community#12555] [Mobile] [answered]
Where is it? I miss it and I need it

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note and suggestion — we don't support repo creation on mobile yet, so for now you'll need to create it on the website first and then it should appear in the app.

**Reply by SkyKingPX** [reply]
Ok, thanks.

**Comment by SkyKingPX** [comment]
But, I really need it ☹️

**Reply by brianlovin** [reply]
For now, I would recommend creating a repository in the browser, either on desktop or mobile, to stay unblocked 😊
